package.json

```diff
@@ -47,6 +47,7 @@
     "eslint": "5.12.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-config-react-app": "^3.0.7",
+    "eslint-config-airbnb": "^17.1.0",
```

Comment:
It looks like this line got added by accident in the rebase. `eslint-config-airbnb` is already being included on line 48.